aggregator/options.go

```diff
@@ -612,7 +698,15 @@ func (o *options) computeFullGaugePrefix() {
 
 // By default we use e.g. ".p50", ".p95", ".p99" for the 50th/95th/99th percentile.
 func defaultTimerQuantileSuffixFn(quantile float64) []byte {
-	return []byte(fmt.Sprintf(".p%0.0f", quantile*100))
+	return []byte(".p" + addSuffixZero(strconv.FormatFloat(quantile, 'f', -1, 64)[2:]))
```

Comment:
I would prefer to do p05 than p5, we need to differentiate quantiles like p05 and p0005. Will add a comment